tests/validation/lib/aws.py

```diff
@@ -271,6 +271,19 @@ def delete_node(self, node, wait_for_deleted=False):
             node = self.wait_for_node_state(node, 'terminated')
         return node
 
+    def delete_eks_cluster(self, cluster_name):
+        ng_names = self._eks_client.list_nodegroups(clusterName=cluster_name)
+        for node_group in ng_names['nodegroups']:
+            print("deleting node group: " + node_group)
+            delete_ng_response = self._eks_client.delete_nodegroup(
+                                                    clusterName=cluster_name,
+                                                    nodegroupName=node_group)
+        waiter_ng = self._eks_client.get_waiter('nodegroup_deleted')
+        for node_group in ng_names['nodegroups']:
```

Comment:
Can we print a wait message here to help debugging if we reach a case where the waiter failed after attempts are exceeded? a print maybe ? or enclosing this in try-except of `WaiterError`.

Reply:
Added a print statement. Boto waiter will throw an error if maximum tries are exceeded.